qemu/tests/virtio_driver_sign_check.py

```diff
@@ -1,85 +1,71 @@
 import logging
-import os
-import re
-import time
 
+from virttest import error_context
 from virttest import utils_misc
+from virttest.utils_windows import virtio_win
 
 
+def get_driver_file_path(session, params):
+    """
+    Get tested driver path and drive_letter,
+    Path like "E:\\vioscsi\\2k8\\x86\\vioscsi.cat/sys/inf"
+    or "A:\\i386\\Win2008\\vioscsi.cat".
+    drive_letter like "E:" or "A:".
+
+    :param session: VM session
+    :param params: Dictionary with the test parameters
+    """
+    driver_path = params["tested_driver"]
+    media_type = params["virtio_win_media_type"]
+    get_drive_letter = getattr(virtio_win, "drive_letter_%s" % media_type)
+    drive_letter = get_drive_letter(session)
+    get_product_dirname = getattr(virtio_win,
+                                  "product_dirname_%s" % media_type)
+    guest_name = get_product_dirname(session)
+    get_arch_dirname = getattr(virtio_win, "arch_dirname_%s" % media_type)
+    guest_arch = get_arch_dirname(session)
+    path = ("{letter}\\{driver}\\{name}\\{arch}\\" if media_type == "iso"
+            else "{letter}\\{arch}\\{name}\\{driver}").\
```

Comment:
Not suggest to use the trailing `\` to combine the two lines, you can either use `()` or separate them into two statements instead.

Reply:
thanks for you advise, updated with ().